### ISSUE TITLE
Bug: 'In file search': If note is 'pinned', the command 'In file search' doesn't work ...

### ISSUE THREAD

**Issue by avirosso** [tadashi-aikawa/obsidian-another-quick-switcher#231]
Version: 11.2.1 (and Obsidian Version 1.5.8): 

If I use the command 'In file search' in a pinned Note, the following happens:

No search modal opens

Instead a new, empty tab opens

and the following error message is in DevTools:
```
Uncaught (in promise) TypeError: Cannot read properties of null (reading 'getValue')
    at InFileModal.indexingItems (plugin:obsidian-another-quick-switcher:5263:52)
    at InFileModal.init (plugin:obsidian-another-quick-switcher:5183:16)
    at showInFileDialog (plugin:obsidian-another-quick-switcher:5540:15)
    at Object.checkCallback (plugin:obsidian-another-quick-switcher:5624:9)
    at Jq (app.js:1:1792093)
    at e.executeCommand (app.js:1:1793657)
    at e.onTrigger (app.js:1:1155885)
    at e.handleKey (app.js:1:748623)
    at e.handleKey (app.js:1:748736)
    at t.handleKey (app.js:1:748945)
```

**Comment by tadashi-aikawa** [comment]
Thank you, @avirosso :)

I could reproduce it 👍

**Comment by tadashi-aikawa** [comment]
@avirosso 
Released in [v11.2.2](https://github.com/tadashi-aikawa/obsidian-another-quick-switcher/releases/tag/11.2.2) 🚀

**Comment by avirosso** [comment]
Thanks! (it works with 11.2.2)
